fdbserver/workloads/RestoreBackup.actor.cpp

```diff
@@ -99,6 +101,7 @@ struct RestoreBackupWorkload final : TestWorkload {
 	}
 
 	ACTOR static Future<Void> _start(RestoreBackupWorkload* self, Database cx) {
+		self->agentFuture = self->backupAgent.run(cx, &self->backupPollDelay, CLIENT_KNOBS->SIM_BACKUP_TASKS_PER_AGENT);
```

Comment:
This will only be necessary in restarting tests, so maybe it should be parameterized. What happens if we attempt to run a backup + restore without restarting? Since this actor is meant to be a general use actor, I'm afraid we'll run into problems if somehow two backup agents are running.

Reply:
Yes, this will addressed as part of https://github.com/apple/foundationdb/issues/3972. For now, this workload is only used as part of restarting tests.